packages/gluestick/src/autoUpgrade/__tests__/index.test.js

```diff
@@ -0,0 +1,68 @@
+jest.mock('fs', () => ({
```

Comment:
I think we should start changing the `index.js` pattern. We probably should only use `index.js` to export the main entry point for each folder. It is specially useful for:

- Looking for files
- Jest normal and watch mode pattern, multiple `index.test.js` make this very hard right now